en/dev_log/ulog_file_format.md

```diff
@@ -395,7 +395,7 @@ struct message_logging_tagged_s {
   uint8_t log_level;
   uint16_t tag;
   uint64_t timestamp;
-  char message[header.msg_size-9]
+  char message[header.msg_size-11]
```

Comment:
From the documentation
> `msg_size` is the size of the message in bytes without the header.

The message below substract 9 because `uint8_t` and `uint64_t` together gives a 9 bytes size.
The [python implementation use 9](https://github.com/PX4/pyulog/blob/7819181f2c9cb0ecfbcd73d46571b0afa44f0d97/pyulog/core.py#L637).
> ```c
> struct message_logging_s {
>   struct message_header_s header; // msg_type = 'L'
>   uint8_t log_level;
>   uint64_t timestamp;
>   char message[header.msg_size-9]
> };
> ```

The message logging tagged has an additional `uint16_t` field that gives the correct size of 11 to substract.
The [python implementation use 11](https://github.com/PX4/pyulog/blob/7819181f2c9cb0ecfbcd73d46571b0afa44f0d97/pyulog/core.py#L664).

Reply:
Thank you! Looks good to me. FYI @bkueng